aip/general.md

```diff
@@ -1,4 +1,7 @@
 ---
+aip_index:
+  order: 10
```

Comment:
This is an interesting idea. It strikes me as something I would have liked to avoid, but that may be ultimately necessary.